templates/types/streaming/fastapi/README-template.md

```diff
@@ -21,10 +21,16 @@ Second, generate the embeddings of the documents in the `./data` directory (if t
 poetry run generate
 ```
 
-Third, run the development server:
+Third, if you want to use the app with a chat UI, you will need to build the frontend once. If not, you can skip this step:
 
 ```
-python main.py
+poetry run build
+```
+
+Finally, run the app:
+
+```
+poetry run dev
```

Comment:
how about `poetry run dev` also runs the dev frontend (via a proxy to `npm run dev`)? then we don't need this confusing change in the readme

Reply:
also don't forgeet to use `poetry run dev` in the other READMEs